iot-hub/Samples/device/ImportExportDevicesSample/Program.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.RetryPolicies;
+
+namespace ImportExportDevices
+{
+    public class Program
+    {
+        //user variables -- add a config file and put these in settings? 
+
+        //IoT Hub connection string. You can get this from the portal.
+        // Log into https://azure.portal.com, go to Resources, find your hub and select it.
+        // Then look for Shared Access Policies and select it. 
+        // Then select IoThubowner and copy one of the connection strings.
+        public static string IoTHubConnectionString = "<connection string to an existing IoT hub>";
```

Comment:
It's a very bad idea security-wise to add any kinds of secrets within the binary. 
Please follow our Program.cs template. This sample would fail the build system of this repo the way it's written.

Reply:
I have no idea what your Program.cs template is. I'll e-mail you because I don't understand most of what you're asking me to do.

Reply:
See all other samples in the same folder.